src/components/themes/Themes.jsx

```diff
@@ -1,13 +1,13 @@
 import themesData from "../../data/themesData";
-
+import './Themes.css'
 export default function Theme() {
   return (
-    <section class="">
+    <section class="grid">
       {themesData.map((el) => (
-        <div class="">
-          <img src={el.themeImageURL} />
+        <div class="theme_item">
```

Comment:
If possible, try to use TailwindCSS for styling instead of the `theme_item` css selector, as it will be easier to maintain later or modify for future websites. Try to use raw CSS as less as possible.